### ISSUE TITLE
Conditions

### ISSUE THREAD

**Issue by paaljoachim** [Desertsnowman/Caldera-Forms#220]
Some more info is needed to grasp how to use the Conditions area.
Clicking Add Condition opens a blank line to where some can do something really really great.... I just need the magic formula..:)

**Comment by oga23** [comment]
Yeah, i realy don't get these variables...i learned to work with conditions well, but these variables and processors thats what i dont get. Is it something to speed up a form maybe?
